kaiax/gasless/impl/getter.go

```diff
@@ -197,9 +197,13 @@ func (g *GaslessModule) IsExecutable(approveTxOrNil, swapTx *types.Transaction)
 		if approveArgs.Amount.Cmp(swapArgs.AmountIn) < 0 {
 			return false
 		}
+		// SP3.
+		if approveTxOrNil.Nonce()+1 != swapTx.Nonce() {
+			return false
+		}
```

Comment:
We should also check from statedb. Please add it to GaslessModule if needed.

Reply:
fix at https://github.com/kaiachain/kaia/pull/263/commits/b352df52dc3b604305f212343f1f79b47773aea7